### ISSUE TITLE
Crash while opening

### ISSUE THREAD

**Issue by teemue** [tirkarthi/NotificationDictionary#6]
The app (ver 0.0.10) won't start at all. Android 11

09-12 12:13:03.716 E/AndroidRuntime(9777): android.view.InflateException: Binary XML file line #17 in com.xtreak.notificationdictionary:layout/card_layout: Binary XML file line #17 in com.xtreak.notificationdictionary:layout/card_layout: Error inflating class androidx.cardview.widget.CardView

09-12 12:13:03.716 E/AndroidRuntime(9777): Caused by: android.view.InflateException: Binary XML file line #17 in com.xtreak.notificationdictionary:layout/card_layout: Error inflating class androidx.cardview.widget.CardView

09-12 12:13:03.716 E/AndroidRuntime(9777): 	at com.xtreak.notificationdictionary.RoomAdapter.onCreateViewHolder(RoomAdapter.kt:43)

**Comment by tirkarthi** [comment]
Thanks for the report. I just added Sentry and saw the crash too. 

* Is this a fresh installation causing issue?
* If it's an update was the previous version working fine?
* Can you please try to clear data for and restart the app to see if it solves the issue?

It seems to be an issue in creating the meaning box. I am on Oreo and was unable to reproduce the issue. I will try to check this on an Android 11 device.

**Comment by tirkarthi** [comment]
Ok, I can reproduce this on Android 11 emulator. This seems to happen when dark mode is on in mobile.

**Comment by teemue** [comment]
Yes you're right - If i turn off Android dark mode, it works!

Completely new installation.

**Comment by tirkarthi** [comment]
I have 2 themes for Light and dark mode in. They have 4 custom attributes to build word and meaning card colors. When dark mode is on it seems that Android uses values-night folder which had been missing the 4 values and in dark mode the lookup causes issue. 

Also to note, values-night was missing the whole dark theme I created in normal mode to be used for toggle so I guess Android has a inheritance level of values-night -> values that in this case was picked up values-night.

Quite a tricky issue since all my emulators, devices and pre-reporting builds from Android even on Android 11 used light mode. I have verified the crash being fixed in https://github.com/tirkarthi/NotificationDictionary/commit/760efef19f44cc0457e4ed3d31689525b469f98c and made a release. I will close the issue once the release is approved.
